vmdb/app/models/server_role.rb

```diff
@@ -52,23 +52,23 @@ def self.to_role(server_role)
   end
 
   def self.all_names
-    self.find(:all, :order => :name, :select => "id, name").collect { |r| r.name }
+    self.order(:name).pluck(:name)
```

Comment:
Good point, it's probably better to put it back when it fails then to proactively put possibly unused selects.